services/ui_backend_service/data/refiner/refinery.py

```diff
@@ -84,3 +83,41 @@ async def _postprocess(self, response: DBResponse):
 
         return DBResponse(response_code=response.response_code,
                           body=body)
+
+
+def unpack_processed_value(value) -> Tuple[bool, str, Any]:
```

Comment:
is the use of `Any` justified here? from the failure examples it looks like the last two returns should both be `Optional(str)` ?

Reply:
`Tuple[bool, Optional[Any], Optional[Any]]` is preferred here since `value` and `details` can be anything.